python/Ganga/Core/MonitoringComponent/Local_GangaMC_Service.py

```diff
@@ -33,6 +33,7 @@
 
 heartbeat_times = None
 global_start_time = None
+global_count = 0
```

Comment:
Frankly If this is showing up more than 5 times in tests (which it isn't at the moment) we will get swamped with errors making life difficult.
I'll amend but I'm not convinced it's needed.